magenta-lib/src/main/scala/magenta/deployment_type/Action.scala

```diff
@@ -0,0 +1,14 @@
+package magenta.deployment_type
+
+import magenta.{DeployTarget, DeploymentPackage, DeploymentResources}
+import magenta.tasks.Task
+
+trait ActionRegister {
+  def add(action: Action)
```

Comment:
that would be nice to have a return type here. I can guess it's a side effect, but having `Unit` would make it super clear